lib/migration-payloads/validation/errors.js

```diff
@@ -39,6 +39,17 @@ const errors = {
     },
     NO_TYPE_CHANGE: (fieldId, ctId, parentFieldType, fieldType) => {
       return `Cannot change the type of field "${fieldId}" on content type "${ctId}" from "${parentFieldType}" to "${fieldType}". Field types cannot be changed.`;
+    },
+    validations: {
+      DUPLICATED_VALIDATION: (duplicatedValue) => {
+        return `A field can't have duplicates in the validations array. Duplicated "${JSON.stringify(duplicatedValue)}"`;
+      },
+      INVALID_VALIDATION_PROPERTY: (propName) => {
+        return `A field can't have "${propName}" as a validation.`;
+      },
+      INVALID_VALIDATION_PARAMETER: (propName, expectedType, actualType) => {
+        return `A fields' "${propName}" validation expected to be "${expectedType}", but got "${actualType}"`;
```

Comment:
I'd change this to `"${propName}" validation expected "${expectedType}", but got "${actualType}"`